_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -423,14 +423,10 @@ mfxStatus VAAPIVideoProcessing::QueryCapabilities(mfxVppCaps& caps)
     }
 #endif
 
-    /* NB! The code below should to be replaced with querying caps from driver*/
-#if defined(LINUX_TARGET_PLATFORM_BXTMIN) || defined(LINUX_TARGET_PLATFORM_BXT)
-    caps.uMaxWidth  = 8192;
-    caps.uMaxHeight = 8192;
-#else
-    caps.uMaxWidth  = 4096;
-    caps.uMaxHeight = 4096;
-#endif
+    MFX_CHECK_WITH_ASSERT(m_pipelineCaps.max_output_width && m_pipelineCaps.max_output_height, MFX_ERR_DEVICE_FAILED);
```

Comment:
In that way we will not work with the Q2 driver release. That's bad. Let's have at least some default if these values are absent. I suggest to use 4K (forget about those things I wrote about BDW and 16K-32 - that's too complex). Just use 4K default since that's actually what most users had before building our code.